### PR TITLE
Cross browser and device fixes

### PR DESCRIPTION
**Pull request by FinalAngel**
- fix firefox CMD + Enter save issue
- fix localstorage on private browsing
- fix config and settings not being saved correctly accross modules
- fix linting issues
- fix font-weight problem on IE for "example.com" menu entry
- fix wizard reload issue when hitting "Enter" (it used to load the same content instead of jumping to the next step)
- fix toolbar toggle on microsoft edge not having the correct background
- fix column widths on pagetree for IE
- fix welcome page markup
- fixed dragging styles in pagetree
- align welcome page logo with djangocms-admin-style
- open new pages in pagetree in "_parent" window instead of "_blank"
- optimise active color in wizard (for selected entries)

### CODE REVIEW

File: cms/static/cms/js/modules/cms.structureboard.js [hunk 1]
```diff
@@ -143,7 +141,7 @@
                 modes.eq(0).on(that.click, function (e) {
                     e.preventDefault();
                     // cancel if already active
-                    if (that.settings.mode === 'structure') {
+                    if (CMS.settings.mode === 'structure') {
```
Comment (vxsx) [inline cms/static/cms/js/modules/cms.structureboard.js]: it actually seems like `state` is a more appropriate name for this object :D

File: cms/static/cms/js/modules/cms.toolbar.js [hunk 2]
```diff
@@ -467,7 +465,7 @@ var CMS = window.CMS || {};
                 this.ui.toolbar.css('margin-top', -toolbarHeight);
                 // animate html
                 this.ui.body.removeClass('cms-toolbar-expanded');
-                this.ui.body.animate({ 'margin-top': (this.config.debug) ? 5 : 0 }, speed);
+                this.ui.body.animate({ 'margin-top': (CMS.config.debug) ? 5 : 0 }, speed);
```
Comment (vxsx) [inline cms/static/cms/js/modules/cms.toolbar.js]: 5 is still hardcoded :/

File: cms/static/cms/js/modules/cms.structureboard.js (hunk 1 above)
Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.structureboard.js]: aye but not a part here to "cleanup" names ;)

File: cms/static/cms/sass/cms.pagetree.scss [hunk 3]
```diff
@@ -130,9 +130,15 @@ $row-height: 41px;
     .col-published input {
         display: none;
     }
+    // the tree is recalculated once you set the navigation to be
+    // shown or hidden. In initialization the calculation is off by
+    // 1px so we need to enforce the with for correct placement
```
Comment (vxsx) [inline cms/static/cms/sass/cms.pagetree.scss]: typo wi**d**th

File: cms/static/cms/sass/components/_welcome.scss [hunk 4]
```diff
@@ -49,11 +49,21 @@
         }
     }
 
-    .cms-welcome-logo {
+    .cms-welcome-logo,
+    .cms-welcome-logo:hover {
```
Comment (vxsx) [inline cms/static/cms/sass/components/_welcome.scss]: why the duplicate?

File: cms/static/cms/js/modules/cms.toolbar.js (hunk 2 above)
Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.toolbar.js]: yeah will keep it in mind for upcoming cleanups

File: cms/static/cms/sass/components/_welcome.scss (hunk 4 above)
Comment (FinalAngel) [inline cms/static/cms/sass/components/_welcome.scss]: `.cms-reset` is the culprit for that :(

File: cms/templates/cms/wizards/create.html [hunk 5]
```diff
@@ -19,8 +19,8 @@
             {% include "cms/wizards/includes/form_fields.html" %}
         </div>
         <div class="submit-row">
+            <input type="submit" class="cms-btn cms-btn-action" value="{% trans "Create" %}">
             <button name="wizard_goto_step" class="cms-btn cms-btn-secondary cms-btn-group" value="{{ wizard.steps.prev }}">{% trans "Back" %}</button>
```
Comment (vxsx) [inline cms/templates/cms/wizards/create.html]: let's still add `type="button"`

**Comment by vxsx** [comment]
overall :+1:

**Pull request merged**